Marsiple/App/New Group/AlbumsViewController.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  AlbumsViewController.swift
+//  Marsiple
+//
+//  Created by Matej Korman on 18/07/2018.
+//  Copyright © 2018 Matej Korman. All rights reserved.
+//
+
+import SnapKit
+
+class AlbumsViewController: UIViewController {
+    private let albums = [Album](repeating: Album(images: [#imageLiteral(resourceName: "Image1"), #imageLiteral(resourceName: "Image2"), #imageLiteral(resourceName: "Image3"), #imageLiteral(resourceName: "Image4"), #imageLiteral(resourceName: "Image5"), #imageLiteral(resourceName: "Image1"), #imageLiteral(resourceName: "Image2"), #imageLiteral(resourceName: "Image3"), #imageLiteral(resourceName: "Image4")]), count: 10)
+    private let albumsView = AlbumsView.autolayoutView()
+    
+    init() {
+        super.init(nibName: nil, bundle: nil)
+        setupNavigationBar()
+        setupView()
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+}
+
+extension AlbumsViewController: UITableViewDataSource {
+    func numberOfSections(in tableView: UITableView) -> Int {
+        return 1
+    }
+    
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return albums.count
+    }
+    
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: "UITableViewCell", for: indexPath) as UITableViewCell
+        let photoCollectionView = PhotoCollectionViewController(photos: albums[indexPath.row].photos)
+        cell.addSubview(photoCollectionView.view)
```

Comment:
Lines 37 and 38 are the offending lines that cause the issue you've left a comment for on the main PR page. `cellForRowAt`'s called multiple times, thus it's important to always remember to `reset` and then `configure` the cell.

Reply:
I understand that these are `reusable cells` and I have to remove all `subviews`, but I don't understand why this would cause the described problem.